app-info/metainfo/org.gnome.Software.WebApp_e730d810bf154c7818b460479ea7826140099a7a.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="web-application">
+  <id>org.gnome.Software.WebApp_e730d810bf154c7818b460479ea7826140099a7a.desktop</id>
+  <name>Instagram</name>
+  <launchable type="url">https://www.instagram.com/</launchable>
+  <url type="homepage">https://www.instagram.com/</url>
+  <summary>Instagram</summary>
+  <description>Create an account or log in to Instagram - A simple, fun &amp; creative way to capture, edit &amp; share photos, videos &amp; messages with friends &amp; family.</description>
```

Comment:
"create an account or log into Instagram" should be removed. The rest of the description looks fine, could also be the summary